src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,56 @@
+package lotto.domain;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.stream.Collectors;
+
+import lotto.enums.Rank;
+import lotto.exception.LottoValidationException;
+
+public class Lotto {
+
+	private static final int LOTTO_NUMBERS_LENGTH = 6;
+
+	private final Set<LottoNumber> lottoNumbers;
+
+	public Lotto(List<Integer> lottoNumbers) {
+		if (lottoNumbers.size() != LOTTO_NUMBERS_LENGTH) {
+			throw new LottoValidationException("로또 번호는 " + LOTTO_NUMBERS_LENGTH + "개의 숫자여야 합니다.");
+		}
+
+		this.lottoNumbers = lottoNumbers.stream()
+			.map(LottoNumber::new)
+			.collect(Collectors.toCollection(TreeSet::new));
+	}
```

Comment:
```suggestion
	public Lotto(Set<LottoNumber> lottoNumbers) {
	    this.lottoNumbers = lottoNumbers;
	}
	
	public static Lotto of(Set<LottoNumber> lottoNumbers) {
	    return new Lotto(lottoNumbers);
	}
	
	public static Lotto of(List<LottoNumber> lottoNumbers) { /** 구현 ** / }
	
	public static Lotto of(String lottoNumbers) { /** 구현 ** / }
```
이렇게 표현할 수 있도록 만들면 좋을 것 같아요!